scripts/build-probe-binaries

```diff
@@ -368,12 +378,18 @@ function rhel_build {
 
 	if [ ! -f $RPM ]; then
 		echo Downloading $RPM [RHEL and CentOS]
-		wget -nv --timeout=${URL_TIMEOUT} --tries=${RETRY} $URL
+
+		if [ -z "$ARTIFACTORY_API_KEY" ]; then
+			wget -nv --timeout=${URL_TIMEOUT} --tries=${RETRY} $URL
```

Comment:
Are we sure this step here *Doesn't* pull RHEL rpms as it does today ? Will this fail ? Outcomes?

Reply:
Not sure exactly what you're asking but we don't even try to pull RHEL rpms the old way because there isn't a public site where we can get them from. We do pull CentOS (open source RHEL) but that only goes up to 7 where RHEL is on 8.  For Openshift 4 we need to support the kernels from RHEL8.

Reply:
Your comment explained it. I was worried we were still pulling RHEL the oldway and your fix was to get around some other issue. Now that we weren't ever pulling them, it is not a flag for me.